h2-storage/src/main/java/com/cgi/hexagon/h2storage/order/H2OrderStorage.java

```diff
@@ -61,6 +63,15 @@ public List<Order> findAll() {
         orderRepository.findAll().forEach(order -> result.add(mapper.fromEntity(order)));
         return result;
     }
+
+    public List<Order> getUnclaimed() {
+        List<Order> result = new ArrayList<>();
+        for (AssemblyOrder order : orderRepository.findAll()) {
```

Comment:
I think this was a misunderstanding. I think we have agreed to use the method findByStatus haven't we?